### PR TITLE
Avoid final in core parameter records

### PR DESCRIPTION
**Pull request by christiankral**
This is a proposal for resoving #3970.

### CODE REVIEW

File: Modelica/Magnetic/FundamentalWave/Examples/BasicMachines/InductionMachines/ComparisonPolyphase/IMC_DOL_Polyphase.mo [hunk 1]
```diff
@@ -80,7 +82,7 @@ model IMC_DOL_Polyphase
             40},{80,60}})));
   parameter
     Modelica.Electrical.Machines.Utilities.ParameterRecords.IM_SquirrelCageData
-    aimcData "Induction machine data"
+    aimcData3 "Induction machine data of three phase machine"
```
Comment (dietmarw) [inline Modelica/Magnetic/FundamentalWave/Examples/BasicMachines/InductionMachines/ComparisonPolyphase/IMC_DOL_Polyphase.mo:85]:
```suggestion
    aimcData3 "Induction machine data of a three-phase machine"
```

**Comment by christiankral** [comment]
@dietmarw I re-phrased each of the modified parameter records. Please re-review.

**Review by AHaumer — APPROVED** [review]
fine with me, thanks!

**Review by HansOlsson — APPROVED** [review]
Ok, looks good.

**Comment by christiankral** [comment]
My opinion is yes we can. We have two valid reviews.

**Pull request merged**